app/src/main/java/com/kickstarter/libs/utils/extensions/UriExt.kt

```diff
@@ -295,7 +298,7 @@ private val PROJECT_SURVEY_BACKING_PLEDGE_REDEMPTION = Pattern.compile(
 )
 
 //  /projects/:creator_param/:project_param/backing/redeem
-private val PROJECT_SURVEY_BACKING_REDEEM = Pattern.compile(
+private val PROJECT_BACKING_REDEEM = Pattern.compile(
     "\\A\\/projects(\\/[a-zA-Z0-9_-]+)?\\/[a-zA-Z0-9_-]+\\/backing\\/redeem\\z"
```

Comment:
It is not a survey -at least no longer in case it ever was-, is PM user entry point